source/html/renderer/src/card-elements.ts

```diff
@@ -470,7 +470,7 @@ export class FactSet extends CardElement {
             return this.speak + '\n';
```

Comment:
Still checking with the team, but in the meantime I made the change to honor the default value property. Again thanks for noticing!